src/frontend/sass/splash-screen.scss

```diff
@@ -0,0 +1,44 @@
+@import './mat-colors';
+// Custom theme support
+@import './custom';
+app-root {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  color: $white;
+}
+
+body {
+  background-image: url(/assets/login-bg.jpg);
```

Comment:
Adding this background image to the body of the app causes the whole app to have the background image (see screenshot).
<img width="1678" alt="screen shot 2018-05-31 at 16 30 35" src="https://user-images.githubusercontent.com/2823726/40791871-5d1ed6a0-64f0-11e8-9063-8760ac1752d6.png">

We expect to see <img width="1680" alt="screen shot 2018-04-09 at 14 18 32" src="https://user-images.githubusercontent.com/2823726/40792826-c7389c4a-64f2-11e8-9e96-e62535ab7580.png">

Reply:
Also, the BG image is still present on the login screen element.
<img width="1680" alt="screen shot 2018-05-31 at 16 34 00" src="https://user-images.githubusercontent.com/2823726/40792636-548796f6-64f2-11e8-9cc4-90092abdb1c3.png">